### ISSUE TITLE
What is this error?

### ISSUE THREAD

**Issue by Multowz** [Fushko/gummy#26]
I'm getting "std::_ios_failure" when I try to change brightness or temperature. I'm currently on Linux Mint.
![image](https://user-images.githubusercontent.com/103082425/236333074-9312e113-2ee4-4f43-8967-ac6a6fca268c.png)

**Comment by Fushko** [comment]
Hi, thanks for the report. 

I honestly have no clue. Let's find out.
- Is the daemon running while this error occurs? 
- Do you get the same error whether or not the daemon is running?
- Does the config file at `~/.config/gummyconf.json` exist? Can you paste it here?

**Comment by GrimPixel** [comment]
You probably didn't understand. Have you typed and entered `gummy start` before this?

**Comment by cefer60** [comment]
I have same problem on debian 12
Is necessary install these packages and then work OK
sudo apt install libxcb1-dev libxcb-randr0-dev libxcb-shm0-dev libxcb-image0-dev libsystemd-dev libsdbus-c++-dev libddcutil-dev
But I did not know that there was a configuration file or that the Start parameter was necessary to start. I think there is a lack of documentation

**Comment by Fushko** [comment]
> I have same problem on debian 12 Is necessary install these packages and then work OK sudo apt install libxcb1-dev libxcb-randr0-dev libxcb-shm0-dev libxcb-image0-dev libsystemd-dev libsdbus-c++-dev libddcutil-dev But I did not know that there was a configuration file or that the Start parameter was necessary to start. I think there is a lack of documentation

All this info is present in the README... If you used the DEB package these should be handled automatically.

**Comment by toprak** [comment]
having same issue on arch linux
```$ gummy start
 terminate called after throwing an instance of 'std::runtime_error'
  what():  Lux data not found for ALS at path: "/sys/bus/iio/devices/iio:device5"
```

**Comment by Fushko** [comment]
> having same issue on arch linux
> 
> ```
> $ gummy start
>  terminate called after throwing an instance of 'std::runtime_error'
>   what():  Lux data not found for ALS at path: "/sys/bus/iio/devices/iio:device5"
> ```

Hi, this is another issue. A light sensor was found but without light data. I did not realize this was possible, which is why I don't handle this exception. 

I can place a workaround to ignore the device, but if you can paste the output of the following commands, that would help me out find a proper solution: 
`ls /sys/bus/iio/devices/` 
`ls /sys/bus/iio/devices/iio:device5`

**Comment by toprak** [comment]
@Fushko 
hi this is output of `ls /sys/bus/iio/devices/`
```
$ ls /sys/bus/iio/devices/
iio:device0  iio:device2  iio:device4  trigger0  trigger2  trigger4
iio:device1  iio:device3  iio:device5  trigger1  trigger3  trigger5
```
this is the output of `ls /sys/bus/iio/devices/iio:device5`
```
$ ls /sys/bus/iio/devices/iio:device5
buffer                      in_magn_x_raw                             in_rot_scale
buffer0                     in_magn_y_raw                             name
dev                         in_magn_z_raw                             power
in_magn_hysteresis          in_rot_from_north_magnetic_tilt_comp_raw  scan_elements
in_magn_offset              in_rot_hysteresis                         subsystem
in_magn_sampling_frequency  in_rot_offset                             trigger
in_magn_scale               in_rot_sampling_frequency                 uevent
```

**Comment by Fushko** [comment]
> @Fushko hi this is output of `ls /sys/bus/iio/devices/`
> 
> ```
> $ ls /sys/bus/iio/devices/
> iio:device0  iio:device2  iio:device4  trigger0  trigger2  trigger4
> iio:device1  iio:device3  iio:device5  trigger1  trigger3  trigger5
> ```
> 
> this is the output of `ls /sys/bus/iio/devices/iio:device5`
> 
> ```
> $ ls /sys/bus/iio/devices/iio:device5
> buffer                      in_magn_x_raw                             in_rot_scale
> buffer0                     in_magn_y_raw                             name
> dev                         in_magn_z_raw                             power
> in_magn_hysteresis          in_rot_from_north_magnetic_tilt_comp_raw  scan_elements
> in_magn_offset              in_rot_hysteresis                         subsystem
> in_magn_sampling_frequency  in_rot_offset                             trigger
> in_magn_scale               in_rot_sampling_frequency                 uevent
> ```

Thanks. Fixed in 0.5.4.

**Comment by toprak** [comment]
@Fushko works on my end now, thanks!
